rifle.py

```diff
@@ -26,28 +27,61 @@ def __init__(self, universe, DRAWING_SCALE):
         self.current_text_ = ""
         self.enemy_color_ = RED
         self.font_size_ = 15
+        self.ids_for_target_tags_ = dict() 
         self.main_character_color_ = GREEN
-        self.target_tags_ = dict()
+        self.projectile_color_ = WHITE 
         self.target_tag_y_spacing_ = 5
+        self.target_tags_ = dict()
         self.targeting_terminal_ = TargetingTerminal(DRAWING_SCALE)
         self.text_antialias_ = 1
         self.ui_font_ = pygame.font.SysFont("monospace", self.font_size_*DRAWING_SCALE)
         self.word_generator_ = WordGenerator()
         self.word_length_min_ = 3
-        self.word_length_range = 3
-
-
+        self.word_length_range_ = 3
+        
+    def get_target_id(self,terminal_input): 
+        for enemy in self.universe_.enemies(): 
+            if terminal_input == self.target_tags_[enemy.ID_]:
+                target_ID = self.ids_for_target_tags_[terminal_input]
+                break
+            else:       
+                target_ID = None
+        return target_ID  
+
+    def check_if_input_matches_target_tag(self): 
```

Comment:
This method name with `check` implies that the return is a bool, but this is not the case. I would suggest `get_target_position(self, target_tag)` and have `self.current_text_` get passed in for the `target_tag` argument.